Automattic/wp-calypso
# Stats: Update iconography for promoting pages on the traffic page

Part of #71733. cc @paulopmt1.

#### Proposed Changes

|Before|After|
|-|-|
|<img width="720" alt="image" src="https://user-images.githubusercontent.com/4044428/211913230-e4502f8c-768e-4de0-8850-41fddb7178cd.png">|<img width="728" alt="image" src="https://user-images.githubusercontent.com/4044428/211912861-2b766b70-d488-4b77-9112-1884cbdb2233.png">|
|<img width="445" alt="image" src="https://user-images.githubusercontent.com/4044428/211682559-4d239a8a-2db7-4814-be85-5de7891cd595.png">|<img width="450" alt="image" src="https://user-images.githubusercontent.com/4044428/211912959-4602cc82-2572-43d6-a81b-8cc5b210f6eb.png">|

* Adds a "blaze" icon to a new "icons" component folder.
* Moves "eye" icon into the "icons" component folder.
* Replaces megaphone icons with the "blaze" icon.

#### Testing Instructions



* Open the live branch for this PR.
* Navigate to the traffic page for a public WordPress.com site.
* Hover over an item in the "Posts and pages" section.
* Ensure you see the new flame icon.
* Ensure that the "eye" icon on the traffic page continues to work as expected for the 7-day highlights and the chart tabs.
> <img width="1256" alt="image" src="https://user-images.githubusercontent.com/4044428/211683930-e982ff91-5e75-47c3-be35-d13160a54925.png">
> 
> <img width="1287" alt="image" src="https://user-images.githubusercontent.com/4044428/211683959-aab00c94-d414-4709-bad9-d039efc36437.png">

#### Pre-merge Checklist
